app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,80 @@
+require 'pry'
+class TweetsController < ApplicationController
+
+
+  get '/tweets' do
+    if session[:user_id]
+      @tweets = Tweet.all 
+      erb :'/tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do 
+    if session[:user_id]
+      erb :'/tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == ""
+      redirect to "/tweets/new"
+    else
+      user = User.find_by_id(session[:user_id])
```

Comment:
we have access to current_user for line 26 and 27
lets use `:user_id => current_user.id`